packages/odyssey-storybook/src/components/odyssey-labs/AppTile/AppTile.stories.tsx

```diff
@@ -317,3 +326,41 @@ export const CustomContent: StoryObj<AppTileProps> = {
     );
   },
 };
+
+export const Loading: StoryObj<AppTileProps> = {
+  render: function C() {
+    const [isDrawerOpen, setIsDrawerOpen] = useState<boolean>(false);
+
+    return (
+      <>
+        <Drawer
+          isOpen={isDrawerOpen}
+          onClose={() => setIsDrawerOpen(false)}
+          showDividers={false}
+        />
```

Comment:
Since Odyssey is providing examples for other teams, we should memoize the function passed to `onClose` as well. Performance doesn't matter in Storybook, but that's going to be more representative of what people are using in production.

Reply:
There are other event handlers we should memoize with `useCallback` as well in this story. Please fix those too :)